pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -61,7 +61,8 @@ func (d *stateData) Clone() framework.StateData {
 // In the Filter phase, pod binding cache is created for the pod and used in
 // Reserve and PreBind phases.
 type VolumeBinding struct {
-	Binder scheduling.SchedulerVolumeBinder
+	Binder                               scheduling.SchedulerVolumeBinder
```

Comment:
Another followup item before beta. We may need changes in other volume related scheduler plugins, like VolumeAttachLimit

Reply:
Added to https://github.com/kubernetes/enhancements/issues/1698#issuecomment-655344680